<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntheticLidarScannerWidget</class>
 <widget class="QMainWindow" name="SyntheticLidarScannerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Theta Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMinThetaAngle"/>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMaxThetaAngle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number of Phi Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtNumberOfPhiPoints"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Phi Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMinPhiAngle"/>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMaxPhiAngle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of Theta Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtNumberOfThetaPoints"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnPreview">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScan">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHideBox">
        <property name="text">
         <string>Hide Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnShowBox">
        <property name="text">
         <string>Show Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblScanning">
        <property name="text">
         <string>Scanning...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QVTKWidget" name="qvtkWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFullOutput"/>
    <addaction name="actionSavePoints"/>
    <addaction name="actionSavePTX"/>
    <addaction name="actionSaveMesh"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveFullOutput">
   <property name="text">
    <string>Save full output</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="text">
    <string>Save points</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSavePTX">
   <property name="text">
    <string>Save PTX</string>
   </property>
  </action>
  <action name="actionSaveMesh">
   <property name="text">
    <string>Save Mesh</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
